setup.py

```diff
@@ -70,7 +68,7 @@
 
 setup(
     name="sdv",
-    version="0.10.1",
+    version="0.11.0",
```

Comment:
Shouldn't be a "0.10.2" enough? I don't see new features or even incompatibilities ...

Reply:
Especially because of protobuf 3 --> 4

Reply:
Ok, but does that mean a "big change" for our users? Or: Do they even recognize?

Reply:
Yes, but isn't that just needed for the stub generation - or generally during "runtime"?